extensions/data-transfer/portability-data-transfer-google/src/test/java/org/datatransferproject/datatransfer/google/photos/GooglePhotosExporterTest.java

```diff
@@ -308,6 +314,36 @@ public void onlyExportAlbumlessPhoto()
         .containsExactly(albumlessPhotoUri + "=d"); // download
   }
 
+  @Test
+  public void photoModelConversionErrorsSkipped() throws Exception {
+    // These media items fail when converting because MediaItem.isPhoto() returns false.
+    GoogleMediaItem failedGoogleMediaItem1 = setUpSinglePhoto("uri1", "failed-photo-1");
+    failedGoogleMediaItem1.setMediaMetadata(new MediaMetadata());
+
+    GoogleMediaItem failedGoogleMediaItem2 = setUpSinglePhoto("uri2", "failed-photo-2");
+    failedGoogleMediaItem2.setMediaMetadata(new MediaMetadata());
+
+    GoogleMediaItem[] mediaItems = {
+        setUpSinglePhoto("uri3", "photoId1"),
+        failedGoogleMediaItem1,
+        setUpSinglePhoto("uri4", "photoId2"),
+        setUpSinglePhoto("uri5", "photoId3"),
+        failedGoogleMediaItem2,
+        setUpSinglePhoto("uri6", "photoId4")
+    };
+
+    MediaItemSearchResponse mediaItemSearchResponse = new MediaItemSearchResponse();
+    mediaItemSearchResponse.setMediaItems(mediaItems);
+    mediaItemSearchResponse.setNextPageToken(null);
+
+    when(photosInterface.listMediaItems(any(), any())).thenReturn(mediaItemSearchResponse);
+
+    ExportResult<PhotosContainerResource> result =
+        googlePhotosExporter.exportPhotos(authData, Optional.of(new IdOnlyContainerResource("albumId")), Optional.empty(), uuid);
+
+    assertThat(result.getExportedData().getPhotos()).hasSize(4);
```

Comment:
nit: good enough. But can we assert ids instead of count?

Reply:
done!